packages/falter-common/files-common/sbin/uci2ffwizard

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+
+#
+#   This script parses all important values into a json file. With this
+#   file one can restore the complete basic configuration of a freifunk
+#   berlin router.
+#
+#   It originates from Freifunk Berlin and is licensed under
+#   GNU General Public License v3.0 or later
+#   Copyright (C) 2023   Martin Hübner
+#
+
+# set -e
+# shellcheck shell=dash
+# shellcheck disable=SC2086
+# shellcheck disable=SC2154
+
+# shellcheck source=/dev/null
+. /lib/functions.sh
+# shellcheck source=/dev/null
+. /usr/share/libubox/jshn.sh
+
+# get contact and community data
+uci_load freifunk
+name="$(uci_get freifunk contact name)"
+nick="$(uci_get freifunk contact nickname)"
+mail="$(uci_get freifunk contact mail)"
+phone="$(uci_get freifunk contact phone)"
+homepage="$(uci_get freifunk contact homepage)" # whitespace-separated, with single quotes, if string contains whitspace
+loc_description="$(uci_get freifunk contact location)"
+com_name="$(uci_get freifunk community name)"
+
+# collect nodes location
+uci_load system
+longitude="$(uci_get system @system[-1] longitude)"
+latitude="$(uci_get system @system[-1] latitude)"
+
+# if contact field contains a config-wizard URL, write it into URL-field
+if case "$mail" in
+    https://config.berlin.freifunk.net/contact*) true ;;
+    *) false ;;
+esac then
+    url="$mail"
+    mail=""
+fi
+
+# load IP-Addresses
+uci_load ffwizard
+share="$(uci_get ffwizard settings sharenet)"
+download="$(( $(uci_get ffwizard settings usersBandwidthDown) * 1000))"
+upload="$(( $(uci_get ffwizard settings usersBandwidthUp) * 1000))"
+monitoring="$(uci_get ffwizard settings enableStats)"
+mesh1="$(uci_get ffwizard settings meship_radio0)"
+mesh2="$(uci_get ffwizard settings meship_radio1)"
+dhcp="$(uci_get ffwizard settings dhcpmesh)"
+
+# autoupdate-stuff
+uci_load autoupdate
+# swap value, as we have different properties here.
+autoupdate_enabled="$(( ! $(uci_get autoupdate cfg disabled) ))"
+
+
+# tunnel-stuff
+uci_load ffberlin-uplink
+uplink_type=$(uci_get ffberlin-uplink preset current)
+if [ "$uplink_type" = tunnelberlin_tunneldigger ]; then
+    is_tunneldigger="tunneldigger"
+else
+    is_tunneldigger=""
+fi
+
+has_bbbdigger="$(uci_get tunneldigger bbbdigger interface)"
+
+# get hostname
+json_load "$(ubus call system board)"
+json_get_var hostname hostname
+
+
+#############################
+#                           #
+#   construct json-string   #
+#                           #
+#############################
+
+json_init
+
+    json_add_string "created" "$(date -u +"%Y-%m-%dT%H:%M:%S+00:00")"
+    json_add_string "version" "1.1"
```

Comment:
I would use version "1.0" since this is the first piece of code that uses the format. The only change I see is that we use arrays for the contact information. We should simply change the falter-ffwizard_spec repo to reflect this change. Another solution would be to use "0.9" which would reflect that the json output does not extract the interfaces.

Reply:
I would prefer the first solution for the ffwizard_spec-repo. The interfaces aren't mandatory in the json anyway, so te resulting json should be a valid v1.0-json

Reply:
Have a look at https://github.com/freifunk-berlin/falter-ffwizard_spec/pull/5

Reply:
ok. 1.0 it is.